api/v1alpha1/openapi.yaml

```diff
@@ -2286,6 +2286,9 @@ components:
           additionalProperties:
             type: string
           description: 'Properties set by the service.'
+        resourceVersion:
+          type: string
+          description: An opaque string that identifies the server's internal version of an object.
```

Comment:
should we document that this is intended to be unique on a fleet level vs globally?

Reply:
Nmd I guess it's self documenting :)